src/docs/sdk/performance/index.mdx

```diff
@@ -117,7 +117,8 @@ tree as well as the unit of reporting to Sentry.
   - `Span` should have a method `startChild` which creates a new span with the current span's id as the new span's `parentSpanId` and the current span's `sampled` value copied over to the new span's `sampled` property
   - The `startChild` method should respect the `maxSpans` limit, and once the limit is reached the SDK should not create new child spans for the given transaction.
   - `Span` should have a method called `toSentryTrace` which returns a string that could be sent as a header called `sentry-trace`.
-  - `Span` should have a method called `iterHeaders` (adapt to platform's naming conventions) that returns an iterable or map of header names and values. This is a thin wrapper containing `return {"sentry-trace": toSentryTrace()}` right now. See `continueFromHeaders` as to why this exists and should be preferred when writing integrations.
+  - `Span` should have a method called `toW3CTrace` which returns a string that could be sent as a header called `traceparent`.
```

Comment:
side note: These docs here assume our SDKs create spans themselves. This is no longer the case in Otel-based SDKs (e.g. JS Node), so we can't define most of these methods on a span/transaction interface. Instead we expose top level helper functions. I think it's fine to keep things as-is in this PR but we probably wanna rework this page a bit to keep things up to date.